safe_control_gym/controllers/lqr/ilqr.py

```diff
@@ -21,6 +21,7 @@ def __init__(
             # Model args
             q_lqr: list = None,
             r_lqr: list = None,
+            prior_info: dict={},
```

Comment:
If you leave prior_info in kwargs and call super()__init__ it will set it. I changed baseController slightly in my own PR to set a default to prior prop as well. Not sure if this is the best way to do it, but it means it doesnt need to be added explicitly to each controller and is handled completely by BaseController. We can chat

Reply:
I agree ideally we don't need to add the `prior_info` argument to every controller's __init__ args. I looked at Federico's PR, that works. Also even without any changes the current `get_prior()` will use the correct defaults but may not be explicit to the user unless they check out the function.